### ISSUE TITLE
New Issue with Nuxt 3.7 - [nuxt] Could not resolve id virtual:pwa-info

### ISSUE THREAD

**Issue by SomethingNew71** [vite-pwa/nuxt#61]
```
[nuxt] Could not resolve id virtual:pwa-info /path/to/project/locally
```

<img width="778" alt="Screenshot 2023-08-25 at 8 35 16 PM" src="https://github.com/vite-pwa/nuxt/assets/4572799/b81ff91a-3c9b-4c65-baa0-ddeaa9cdf05d">

**Comment by userquin** [comment]
https://github.com/nuxt/nuxt/issues/22833

**Comment by userquin** [comment]
I have send a PR to fix it to Nuxt repo.

**Comment by SomethingNew71** [comment]
Thank you!

**Comment by userquin** [comment]
should be resolved in Nuxt 3.7.1: https://github.com/nuxt/nuxt/pull/22824
